src/Microsoft.Health.Fhir.Liquid.Converter/Parsers/JsonDataParser.cs

```diff
@@ -22,7 +42,8 @@ public object Parse(string json)
 
             try
             {
-                return JToken.Parse(json).ToObject();
```

Comment:
Does JObject.Parse(json) also have the same behaviour? If so, I do see other places where JObject.Parse is used and would be good to check if a similar setting needs to be applied there too depending on the input being parsed in those cases.

Reply:
I'd suspect that all JXXX.Parse methods have the same behavior.  There seem to be some default settings in Newtonsoft that are undesirable for our needs. 

Agree that we should go through and address this if needed across the project. I suggest doing that work in a separate PR to reduce the scope of this one, which is to address a known customer issue.

We may also want to use this opportunity to move away from Newtonsoft and onto .Net's implementation, which would be a bigger effort.

Reply:
Makes sense. But just to point out, in the thread you forwarded, Dustin mentioned that this problem exists even in the post processing logic which also impacts the customer's issue being addressed.

Reply:
I think there is a potential issue with 'pre/post' processing the result in ADF, and not within Convert/FHIR Server. Looking at the internal [PostProcessor](https://github.com/microsoft/FHIR-Converter/blob/main/src/Microsoft.Health.Fhir.Liquid.Converter/OutputProcessors/PostProcessor.cs#L48) of Json it looks like DateParsing is already disabled. I've tested out using Convert/Fhir Server directly and verified that results can come back with the original date format preserved.